autobuild/windows/autobuild_windowsinstaller_3_copy_files.ps1

```diff
@@ -27,31 +27,30 @@ if (($jamulus_buildversionstring -eq $null) -or ($jamulus_buildversionstring -eq
     $jamulus_buildversionstring = "NoVersion"
 }
 
+# Amend BuildOption when needed, only needed when the name build option in the artifact needs to be different
+if ($BuildOption -Eq "jackonwindows")
```

Comment:
Should be a switch if possible in PS. We might add more APIs like PortAudio in future

Reply:
Updated in latest commit to use switch command